src/map/itemdb.cpp

```diff
@@ -3061,6 +3061,13 @@ void ItemGroupDatabase::pc_get_itemgroup_sub( map_session_data& sd, bool identif
 				tmp.refine = 0;
 			}
 
+			if (data->maximumEnchantgrade > 0 && data->minimumEnchantgrade <= data->maximumEnchantgrade) {
+				tmp.enchantgrade = static_cast<uint8>(rnd_value<uint16>(data->minimumEnchantgrade, data->maximumEnchantgrade));
+			}
+			else {
+				tmp.enchantgrade = 0;
```

Comment:
```suggestion
				tmp.enchantgrade = ENCHANTGRADE_NONE;
```
Maybe add static_cast, if necessary.